deps/v8/build/toolchain.gypi

```diff
@@ -1104,7 +1104,7 @@
           'VCCLCompilerTool': {
             'Optimization': '0',
             'conditions': [
-              ['component=="shared_library"', {
+              ['component=="shared_library" or node_shared=="true"', {
```

Comment:
Agreed, as it is I don't know if this change to the V8 gyp files can be accepted at all (as a floating patch) since the policy is to not patch upstream dependencies. If we have to submit the changes upstream to V8 first then `force_dynamic_crt` makes much more sense. Thanks.